packages/next/src/server/image-optimizer.ts

```diff
@@ -240,6 +241,10 @@ export class ImageOptimizerCache {
       }
     }
 
+    if (INVALID_URL_REGEX.test(url)) {
```

Comment:
Line 238 already handles `//` which would also handle `///`.

I think you can change this to the following:

```suggestion
    if (url.startsWith('\\') {
```